src/log.component.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import { ExtensionStarterBindings } from './keys';
```

Comment:
coding style - no spaces inside/around curly braces

Reply:
We use `-` for file names, for example, `src/log-component.ts`.